Cargo.toml

```diff
@@ -9,12 +9,18 @@ license = "MIT/Apache-2.0"
 repository = "https://github.com/filecoin-project/rust-gpu-tools"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+default = ["opencl"]
```

Comment:
I think we can default to both

Reply:
The reason I didn't make it the default is that you cannot run the test without having CUDA installed. Although it's true for OpenCL as well, you can easily have OpenCL installed, CUDA is harder to install.

But it's easy enough to disable it, so I guess enabling it by default makes sense.